client/package.json

```
@@ -1,7 +1,7 @@
 {
-  "name": "ecdsa-node",
+  "name": "ecdsa-node-client",
   "private": true,
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Let's have this match the incremented Server version:

```suggestion
  "version": "1.0.1",
```